### ISSUE TITLE
When using screen reader , the radzen tabs cannot be navigated using arrow keys

### ISSUE THREAD

**Issue by truecaller12** [radzenhq/radzen-blazor#2528]
**Describe the bug**
**When using Screen Reader** , and having horizontal tabs ( https://blazor.radzen.com/tabs)   the radzen tabs cannot be navigated using arrow keys ( right and left arrow keys). Only up and down arrow key works. But for Horizontal Tabs, right and left arrow keys should as well navigate between the tabs

**To Reproduce**
1. Go to 'https://blazor.radzen.com/tabs'
2. Click on 'Example tab' or reach the component using Tab. and you will see that Example tab is selected ( blue underline)
3.  After this you cannot reach the 'Edit source' tab. 
4. Clicking on Arrow keys, does nothing. 

**Expected behavior**
When Screen reader is on, arrow key should navigate to next tabs 

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser  : Edge
 - Version 147.0.3912.60 

**Additional context**
